geomesa-tools/conf/application.conf

```diff
@@ -78,7 +91,29 @@ geomesa {
         { name = "vis",      transform = "$9::string"                  }
       ]
     }
-    "example-xml" = {
+    "example-csv-per-attr-visibilities" = {
+      type   = "delimited-text"
+      format = "CSV"
+      options {
+        skip-lines = 1
+      }
+      id-field = "toString($fid)",
+      user-data = {
+        geomesa.feature.visibility = "'owl,owl,owl,newt,newt,owl'"
```

Comment:
btw, I already added a (non-attribute-level) vis example here: https://github.com/locationtech/geomesa/blob/master/geomesa-tools/conf/application.conf#L58
I guess we have multiple places for stuff...